server/backend/api/app/routers/router_user_role_assignment.py

```diff
@@ -19,7 +19,7 @@ def create_user_role_assignment(
     role_assignment_request: schemas.FamUserRoleAssignmentCreate,
     db: Session = Depends(database.get_db),
     token_claims: dict = Depends(jwt_validation.authorize),
-    requester: str = Depends(jwt_validation.get_request_username),
+    cognito_username: str = Depends(jwt_validation.get_request_cognito_username),
```

Comment:
This is likely more accurately called cognito_user_id

Reply:
Changed (get_request_cognito_user_id)